pkg/oc/admin/diagnostics/diagnostics/cluster/scc.go

```diff
@@ -0,0 +1,95 @@
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	kerrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/authorization"
+	authorizationtypedclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/authorization/internalversion"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/types"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/util"
+	policycmd "github.com/openshift/origin/pkg/oc/admin/policy"
+	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	securitytypedclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+)
+
+// ClusterSCC is a Diagnostic to check that the default securitycontextconstraints match expectations
```

Comment:
Consider improving this comment to briefly describe these expectations. Otherwise this comment looks useless and produces more questions rather than making something clear.

P.S. godoc comment should also end with dot AFAIR.

Reply:
I used this comment based on ClusterRole's diagnostics:

https://github.com/openshift/origin/blob/12550a638027fb4b71332162f1d459a9d7d3c1c7/pkg/oc/admin/diagnostics/diagnostics/cluster/roles.go#L21

I will update `roles.go` if updated comment is fine.